docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -84,14 +84,17 @@ SQL support
 -----------
 
 The connector provides **read access and limited write access** to data and
-metadata in Iceberg. The following section lists all supported SQL statements
-and provides further details.
+metadata in Iceberg. The following section lists supported SQL statements and
+provides further details.
 
-  .. important::
+.. important::
```

Comment:
I don't think we need to use "important" here, as it's just information like any other, and having this formatting in every connector will appear cluttered. We don't say "important: here are the requirements", rather just "requirements".

Reply:
In all our feedback on the changes along this that are already live we heard that the fact that no other statements are supported is very important. We originally used note and then upgrade to warning. That was too much though and we found important to work well.

Reply:
None of these changes are live yet (in a released version). I'm not convinced that this is more important than other things in the connector documentation, and having this same "important" note for every connector decreases the value of such notifications.

Reply:
Discussed in call and updated as decided with @dain and @electrum